packages/patternfly-4/react-core/src/components/Card/Card.md

```diff
@@ -21,6 +22,260 @@ SimpleCard = () => (
 );
 ```
 
+## Card with image and actions 
+```js
+import React from 'react'; 
+import { Dropdown, DropdownToggle, DropdownItem, DropdownSeparator, DropdownPosition, DropdownDirection, KebabToggle, Card, CardHead, CardActions, CardHeader, CardBody } from '@patternfly/react-core'; 
+import pfLogo from './examples/pfLogo.svg'; 
+
+class KebabDropdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false,
+      check1: false
+    };
+    this.onToggle = isOpen => {
+      this.setState({
+        isOpen
+      });
+    };
+    this.onSelect = event => {
+      this.setState({
+        isOpen: !this.state.isOpen
+      });
+    };
+    this.onClick = (checked, event) => {
+      const target = event.target; 
+      const value = target.type === 'checkbox' ? target.checked : target.value; 
+      const name = target.name; 
+      this.setState({ [name]: value }); 
+    }; 
+  }
+
+  render() {
+    const { isOpen } = this.state;
+    const dropdownItems = [
+      <DropdownItem key="link">Link</DropdownItem>,
+      <DropdownItem key="action" component="button">
+        Action
+      </DropdownItem>,
+      <DropdownItem key="disabled link" isDisabled>
+        Disabled Link
+      </DropdownItem>,
+      <DropdownItem key="disabled action" isDisabled component="button">
+        Disabled Action
+      </DropdownItem>,
+      <DropdownSeparator key="separator" />,
+      <DropdownItem key="separated link">Separated Link</DropdownItem>,
+      <DropdownItem key="separated action" component="button">
+        Separated Action
+      </DropdownItem>
+    ];
+    return (
+      <Card>
+        <CardHead>
+          <img src={pfLogo} style={{height: "50px"}}/>
+          <CardActions>
+            <Dropdown
+              onSelect={this.onSelect}
+              toggle={<KebabToggle onToggle={this.onToggle} />}
+              isOpen={isOpen}
+              isPlain
+              dropdownItems={dropdownItems}
+              position={'right'}
+            />
+            <Checkbox 
```

Comment:
Can we just use an `<input type="check">` here instead of the check component? The check component is designed to work with a check + label pair. For reference, we have plain checkboxes in the "selectable table" example https://2122-pr-patternfly-react-patternfly.surge.sh/patternfly-4/components/table/

Reply:
I changed it to an input, let me know if that looks ok. :)